etc/nebula-graphd.conf.default

```diff
@@ -36,10 +36,10 @@
 --session_idle_timeout_secs=60000
 # The number of threads to accept incoming connections
 --num_accept_threads=1
```

Comment:
Miss this option?

Reply:
> Miss this option?

Nope. One single thread is enough to handle incomming connections.